entrypoint.sh

```diff
@@ -2,44 +2,44 @@
 
 set -e
 
-if [ -z "${NEXTAUTH_URL}" ]; then
+if [ ! "${NEXTAUTH_URL+x}" ]; then
     echo "Provide the NEXTAUTH_URL env variable"
     exit 1
 fi
 
-if [ -z "${NEXTAUTH_SECRET}" ]; then
+if [ ! "${NEXTAUTH_SECRET+x}" ]; then
     echo "Provide the NEXTAUTH_SECRET env variable"
     exit 1
 fi
 
-if [ -z "${KEYCLOAK_HOST}" ]; then
+if [ ! "${KEYCLOAK_HOST+x}" ]; then
     echo "Provide the KEYCLOAK_HOST env variable"
     exit 1
 fi
 
-if [ -z "${KEYCLOAK_CLIENT_ID}" ]; then
+if [ ! "${KEYCLOAK_CLIENT_ID+x}" ]; then
     echo "Provide the KEYCLOAK_CLIENT_ID env variable"
     exit 1
 fi
 
-if [ -z "${KEYCLOAK_CLIENT_SECRET}" ]; then
+if [ ! "${KEYCLOAK_CLIENT_SECRET+x}" ]; then
     echo "Provide the KEYCLOAK_CLIENT_SECRET env variable"
     exit 1
 fi
 
-if [ -z "${KEYCLOAK_REALM}" ]; then
+if [ ! "${KEYCLOAK_REALM+x}" ]; then
     echo "Provide the KEYCLOAK_REALM env variable"
     exit 1
 fi
 
-if [ -z "${LHUT_API_URL}" ]; then
+if [ ! "${LHUT_API_URL+x}" ]; then
     echo "Provide the LHUT_API_URL env variable"
     exit 1
 fi
 
-if [ -z "${LHUT_TENANT_ID}" ]; then
+if [ ! "${LHUT_TENANT_ID+x}" ]; then
     echo "Provide the LHUT_TENANT_ID env variable"
     exit 1
 fi
 
-/entrypoint.sh
+npm run start -ws --if-present -- --port ${PORT} --hostname ${HOSTNAME}
```

Comment:
I'm not sure about this, this is a production build https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile#L70

npm run is not I guess

any idea @mijailr

Reply:
you are correct @sauljabin I will change it

Reply:
> I'm not sure about this, this is a production build https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile#L70
> 
> npm run is not I guess
> 
> any idea @mijailr

Just FYI npm run start does `next start` which is production and essentially the same as `node server.js`